src/FakerDotNet/Fakers/SpaceFaker.cs

```diff
@@ -0,0 +1,106 @@
+﻿using FakerDotNet.Data;
+using System;
+
+namespace FakerDotNet.Fakers
+{
+    public interface ISpaceFaker
+    {
+        string Planet();
+        string Moon();
+        string Galaxy();
+        string Nebula();
+        string StarCluster();
+        string Constellation();
+        string Star();
+        string Agency();
+        string AgencyAbv();
+        string NasaSpaceCraft();
+        string Company();
+        string DistanceMeasurement();
+        string Meteorite();
+        string LaunchVehicle();
+    }
+
+    internal class SpaceFaker : ISpaceFaker
+    {
+        private static readonly SpaceData Data = new SpaceData();
+        private static readonly NumberFaker NumberFaker = new NumberFaker();
```

Comment:
You can remove this and instead access the `INumberFaker` from `_fakerContainer.Number`. Then you can mock that in the tests :+1: